radiography/src/main/java/radiography/StateRenderer.kt

```diff
@@ -0,0 +1,36 @@
+package radiography
+
+import radiography.StateRenderer.Companion.from
+
+/**
+ * Renders extra attributes for specifics types in the output of [Radiography.scan].
+ * Call [from] to create instances with reified types:
+ * ```
+ *  val myRenderer: StateRenderer<MyView> = from { myView, append ->
```

Comment:
A more idiomatic API and way to call this would be `val myRenderer = stateRendererFor<MyView> { myView, append ->`

`from` is a little too generic for a top-level method IMO. And it _should_ be a top-level method IMO, since there's no reason to namespace it if you're going to static-import anyway.

Reply:
I like the suggested name!

I still think there's a UX difference with namespacing, simply in how the IDE prioritizes auto completes (ie I think going top level will give us a higher priority and I probably wouldn't want that).

I'm not 100% certain though.

Reply:
I'm not sure about that. There was an issue for a while where imports from `object`s didn't show up at all, but I think that's fixed, or will be in 1.4.